app/models/computacenter/backfill_ledger.rb

```diff
@@ -0,0 +1,43 @@
+require 'csv'
+
+module Computacenter
+  class BackfillLedger
+    def call
+      users.each do |user|
+        next if UserChange.where(user_id: user.id).exists?
+
+        UserChange.create!(
+          user_id: user.id,
+          first_name: user.first_name,
+          last_name: user.last_name,
+          email_address: user.email_address,
+          telephone: user.telephone,
+          responsible_body: user.effective_responsible_body.name,
+          responsible_body_urn: user.effective_responsible_body.computacenter_identifier,
+          cc_sold_to_number: user.effective_responsible_body.computacenter_reference,
+          school: user.school&.name,
+          school_urn: user.school&.urn,
+          cc_ship_to_number: user.school&.computacenter_reference,
+          updated_at_timestamp: user.created_at,
+          type_of_update: 'New',
+          original_first_name: nil,
+          original_last_name: nil,
+          original_email_address: nil,
+          original_telephone: nil,
+          original_responsible_body: nil,
+          original_responsible_body_urn: nil,
+          original_cc_sold_to_number: nil,
+          original_school: nil,
+          original_school_urn: nil,
+          original_cc_ship_to_number: nil,
+        )
+      end
```

Comment:
This code looks almost identical to the `Computacenter::Ledger.user_changes` method (similarly the `users` method below) - would it make more sense for this whole model to actually just be a `backfill` method on `Computacenter::Ledger` ?

Reply:
you could then take the code which constructs a UserChange for a given User and make it either a class method on UserChange (something like `UserChange.for_user(user)`) or an instance method on user (something like `user.to_user_change`)

Reply:
`Computacenter::Ledger#user_changes` will be swapped out in the near future. It will probably read some like
```ruby
def user_changes
  Computacenter::UserChange.all
end
```
It is a stop gap as that table isn't available yet so we're reading in users for now.

I can move to `UserChange.for_user(user)` or `user.to_user_change` but didn't want to pollute those classes. Also a `User` can't really be converted to a "change" as a user has a current state but a "change" is a change between states.

I feel like one we have a process in place we can probably delete this whole file

Reply:
OK, that makes sense 👍